spec/unit/utils.spec.ts

```diff
@@ -2,7 +2,7 @@ import * as utils from "../../src/utils";
 import {
     alphabetPad,
     averageBetweenStrings,
-    baseToString,
+    baseToString, deepSortedObjectEntries,
```

Comment:
Maybe on separate lines to match the rest here...? Or else smush the others into something more compact...?